### PR TITLE
Various changes for v1.1.0 release.

### PR DESCRIPTION
**Pull request by OleksiiZubko**
- Update packages version to v1.1.0.
- Adapt CHANGELOG.md with v1.1.0 release changeset.
- Update development dependencies.

Resolves: OLPEDGE-1208

Signed-off-by: Oleksii Zubko <ext-oleksii.zubko@here.com>

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,20 @@
...
+
+**olp-sdk-dataservice-read**
+
+* Added method for fetching partitions metadata for specific partition IDs.
+* Updated documentation for public API
+
+**olp-sdk-authentication**
+
+* Added project support.
+* Updated documentation for public API
```
Comment (andescu) [inline CHANGELOG.md]:
```suggestion
* Updated documentation for public API.
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,20 @@
+## v1.1.0 (11/12/2019)
+
+**Common**
+
+* Updated development dependencies.
+
+**olp-sdk-dataservice-read**
+
+* Added method for fetching partitions metadata for specific partition IDs.
```
Comment (andescu) [inline CHANGELOG.md]: Please specify which method it is, e.g. CPP SDK:

"Added new class VersionedLayerClient that is used to access versioned layers in OLP. Class implements GetData, GetPartitions, PrefetchTiles methods from CatalogClient."

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,20 @@
...
+* Updated documentation for public API
```
Comment (andescu) [inline CHANGELOG.md]:
```suggestion
* Updated documentation for public API.
```

File: CHANGELOG.md (hunk 2 above)
Comment (OleksiiZubko) [reply CHANGELOG.md]: updated

File: CHANGELOG.md (hunk 1 above)
Comment (OleksiiZubko) [reply CHANGELOG.md]: updated

File: CHANGELOG.md (hunk 3 above)
Comment (OleksiiZubko) [reply CHANGELOG.md]: updated

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,3 +1,21 @@
+## v1.1.0 (11/12/2019)
+
+**Common**
+
+* Updated development dependencies.
+
+**olp-sdk-dataservice-read**
+
+* Added method  getPartitionsById to the QueryClient for fetching partitions metadata for specific partition IDs.
```
Comment (andescu) [inline CHANGELOG.md]:
```suggestion
* Added method `getPartitionsById` to the `QueryClient` class for fetching partitions metadata for specific partition IDs.
```

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,3 +1,21 @@
...
+* Updated method getPartitions in VersionedLayerClient and VolatileLayerClient to be able to fetch partitions metadata for specific partition IDs
```
Comment (andescu) [inline CHANGELOG.md]:
```suggestion
* Updated method `getPartitions` in `VersionedLayerClient` and `VolatileLayerClient` to be able to fetch partitions metadata for specific partition IDs.
```

File: CHANGELOG.md [hunk 6]
```diff
@@ -1,3 +1,21 @@
+## v1.1.0 (11/12/2019)
+
+**Common**
+
+* Updated development dependencies.
```
Comment (HalynaDumych) [inline CHANGELOG.md]:
```suggestion
* Updated the list of development dependencies.
```

File: CHANGELOG.md (hunk 4 above)
Comment (HalynaDumych) [reply CHANGELOG.md]:
```suggestion
* Added the `getPartitionsById` method to `QueryClient`. Now, you can fetch metadata from specific partitions using their IDs.
```

File: CHANGELOG.md (hunk 5 above)
Comment (HalynaDumych) [reply CHANGELOG.md]:
```suggestion
* Updated the `getPartitions` method in `VersionedLayerClient` and `VolatileLayerClient`. Now, you can fetch metadata from specific partitions of a volatile or versioned layer using partitions IDs.
```

File: CHANGELOG.md [hunk 7]
```diff
@@ -1,3 +1,21 @@
...
+
+**olp-sdk-dataservice-read**
+
+* Added method  getPartitionsById to the QueryClient for fetching partitions metadata for specific partition IDs.
+* Updated method getPartitions in VersionedLayerClient and VolatileLayerClient to be able to fetch partitions metadata for specific partition IDs
+* Updated documentation for public API.
```
Comment (HalynaDumych) [inline CHANGELOG.md]:
```suggestion
* Updated documentation for the public API.`
```

File: CHANGELOG.md [hunk 8]
```diff
@@ -1,3 +1,21 @@
...
+
+**olp-sdk-authentication**
+
+* Added project support.
+* Updated documentation for public API.
```
Comment (HalynaDumych) [inline CHANGELOG.md]:
```suggestion
* Updated documentation for the public API.
```

File: CHANGELOG.md [hunk 9]
```diff
@@ -1,3 +1,21 @@
+## v1.1.0 (11/12/2019)
+
+**Common**
+
+* Updated development dependencies.
+
+**olp-sdk-dataservice-read**
+
+* Added the getPartitionsById method to QueryClient. Now, you can fetch metadata from specific partitions using their IDs.
```
Comment (HalynaDumych) [inline CHANGELOG.md]:
```suggestion
* Added the `getPartitionsById` method to `QueryClient`. Now, you can fetch metadata from specific partitions using their IDs.
```

File: CHANGELOG.md [hunk 10]
```diff
@@ -1,3 +1,21 @@
...
+* Updated the getPartitions method in VersionedLayerClient and VolatileLayerClient. Now, you can fetch metadata from specific partitions of a volatile or versioned layer using partitions IDs.
```
Comment (HalynaDumych) [inline CHANGELOG.md]:
```suggestion
* Updated the `getPartitions` method in `VersionedLayerClient` and `VolatileLayerClient`. Now, you can fetch metadata from specific partitions of a volatile or versioned layer using partitions IDs.
```

File: CHANGELOG.md [hunk 11]
```diff
@@ -1,3 +1,21 @@
...
+* Updated documentation for public API.
```
Comment (HalynaDumych) [inline CHANGELOG.md]:
```suggestion
* Updated documentation for the public API.
```

File: CHANGELOG.md (hunk 6 above)
Comment (HalynaDumych) [inline CHANGELOG.md]:
```suggestion
* Updated the development dependencies.
```

File: CHANGELOG.md [hunk 12]
```diff
@@ -1,3 +1,21 @@
...
+
+**Common**
+
+* Updated development dependencies.
+
+**olp-sdk-dataservice-read**
+
+* Added the getPartitionsById method to QueryClient. Now, you can fetch metadata from specific partitions using their IDs.
+* Updated the getPartitions method in VersionedLayerClient and VolatileLayerClient. Now, you can fetch metadata from specific partitions of a volatile or versioned layer using partitions IDs.
+* Updated documentation for public the API.
```
Comment (HalynaDumych) [inline CHANGELOG.md]:
```suggestion
* Updated documentation for the public API.
```

File: CHANGELOG.md [hunk 13]
```diff
@@ -1,3 +1,21 @@
...
+
+**olp-sdk-authentication**
+
+* The scope support was added to `UserAuth` through `UserAuthConfig`. You can now access the project bound resources using the `UserAuthConfig` scope field.
+* Updated documentation for public the API.
```
Comment (HalynaDumych) [inline CHANGELOG.md]:
```suggestion
* Updated documentation for the public API.
```

**Pull request merged**